packages/hebao_v1/contracts/modules/core/WalletFactory.sol

```diff
@@ -58,18 +61,36 @@ contract WalletFactory is ReentrancyGuard
         allowEmptyENS = _allowEmptyENS;
     }
 
-    function computeWalletAddress(
```

Comment:
We need this function because we must get the wallet address ahead of its creation so we can compute the ensApproval.